### ISSUE TITLE
Fix Segment calls user_created and workspace_created

### ISSUE THREAD

**Issue by nataliekwong** [airbytehq/airbyte#12842]
## Current Behavior
There are two Segment calls that have not fired since 5/11.
- user_created
- workspace_created

## Expected Behavior
We see that there are users created and workspaces created since 5/11 using Postgres tables. The Segment events power the following dashboards:

Activation Rate: https://airbyte.metabaseapp.com/dashboard/10-activation-dashboard
GTM Summary: https://airbyte.metabaseapp.com/dashboard/72-gtm-summary

**Comment by nataliekwong** [comment]
@andyjih Is there capacity on Platform to help bring back these Segment calls?

The reason this is urgent on the GTM side is because we don't have an alternative way to find the 'cloud_user_id' that only exists on the Segment calls. This is how we join the frontend Segment calls to the backend (api) ones. 

As an alternative data source, we have Postgres tables syncing to the dwh which includes users, but we only have the 'user_id' in those tables - we don't store the 'cloud_user_id'. Rewriting all our dashboards would a be pretty significant undertaking as it would require rolling everything up to the workspace instead of the user.

**Comment by andyjih** [comment]
@nataliekwong Let me see what we can do next week!

**Comment by edfarraye45** [comment]
Hey @andyjih Do we have any ETA on when this will be fixed? We can't report on any metrics past 5/11 without this.

**Comment by andyjih** [comment]
@edfarraye45 Working on it! Will follow-up when I have an ETA soon.

**Comment by lmossman** [comment]
@nataliekwong are other segment calls working, e.g. "User Invited", "Workspace Updated", etc? Trying to determine if this is issue is limited to only those two specific actions or if this is an issue for all segment calls coming from cloud.

Also, looking at the code it seems like we are setting `cloud_user_id` in the segment call to the value of the `id` column in the `user` table in the cloud postgres database. So I would expect those values to be the same - not sure if the `user_id` you were referencing is something else.

**Comment by andyjih** [comment]
@lmossman `User Invited` also seems to have dropped off on May 11, but `Workspace Updated` is still firing.

**Comment by nataliekwong** [comment]
@lmossman Would this be the right way to check if the `cloud_user_id` in the Segment call is the `id` in the user Postgres table?

An example of the Segment call with the cloud_user_id:
`select id,cloud_user_id from `ab-analytics-308500.api.user_created` where cloud_user_id = 'e4f45853-60f2-4d12-ae79-19e8ef1977c9'`

And looking for that id in the user table:
`select id from airbyte_prod_cloudapi.user where id = 'e4f45853-60f2-4d12-ae79-19e8ef1977c9'`

**Comment by lmossman** [comment]
@nataliekwong yes that looks correct to me

**Comment by nataliekwong** [comment]
@lmossman I ran the second query in BigQuery/Metabase and it's not returning a result when I try to look for that as an id, but lmk if there's a better way to check? 

<img width="1001" alt="Screen Shot 2022-05-17 at 2 34 21 PM" src="https://user-images.githubusercontent.com/38087517/168885768-85826e0e-12fa-44b1-97cf-8eabe8188cde.png">

**Comment by andyjih** [comment]
@malikdiarra Pushed a fix in this PR: https://github.com/airbytehq/airbyte-cloud/pull/1684

I'll close this issue when I've confirmed the fix on prod.

**Comment by andyjih** [comment]
@nataliekwong @edfarraye45 I was able to verify that the `User Created` event was triggered in production. 

<img width="1469" alt="2022-05-17_15-08-08" src="https://user-images.githubusercontent.com/633226/168919116-323c8b6d-faa6-433d-b248-c239fae82428.png">

I'll leave this issue open until you verify in metabase in your dashboards that data appears to be flowing correctly.

Note for the future: in general, I strongly recommend that business critical data not be dependent on events and instead be based on what's in the db/warehouse. Perhaps something to discuss w/ @ChristopheDuong and team later.

**Comment by nataliekwong** [comment]
Yep, I agree with you there @andyjih! All dashboards built off raw data in Metabase should be treated as beta as they were built quickly to get a minimum set of baseline metrics delivered to the company. 

From what @lmossman mentioned earlier, the cloud_user_id should be in Postgres user table as the id, but when I try to query for that it doesn't return anything. 

Happy to refactor dashboards to the Postgres tables (and agree this is the right approach) - I just don't know of a way to do it.

**Comment by nataliekwong** [comment]
Got it! That is very helpful, I got the dashboards switched to the Postgres tables today.

**Comment by nataliekwong** [comment]
No action required. Just something I've noticed which is quite confusing. I pulled the number of events for the user_created Segment call and it looks like nothing lapsed between the time we noticed the issue (5/11 and the resolution of the issue (5/18).
 
![Screen Shot 2022-06-01 at 12 08 06 AM](https://user-images.githubusercontent.com/38087517/171325877-5c34e63d-9f05-4e22-87c5-80c99d76bfab.png)

> select
> cast(sent_at as date) as sent_date,
> count(*) as users_created
> from airbyte_ui_cloud.airbyte_ui_user_created
> where sent_at >= '2022-05-01'
> group by 1 order by 1 desc

I did similar pulls to aggregate on received_at, original_timestamp, and they all show the same thing, like we never stopped receiving the events.
